Mathlib/Topology/Order/Category/FrameAdjunction.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy
+-/
+import Mathlib.Topology.Category.Locale
+
+/-!
+# Adjunction between Locales and Topological Spaces
+
+This file defines functors between the categories of Locales and Topological Spaces
+and proves that they form an adjunction.
+
+## Main declarations
+
+* `pt`: the *points* functor from the category of locales to the category of topological spaces.
```

Comment:
```suggestion
* `pt`: the *points* functor from the category of Locales to the category of Topological Spaces.
```

Reply:
Sorry I don't think that makes sense to capitalise non-Lean names.

Reply:
Agreed, I will turn the first occurrence into lower case and leave these lower case.